view/base/web/capture.js

```diff
@@ -183,6 +198,10 @@ requirejs(['jquery', 'mage/url','domReady'], function($, urlBuilder) {
             evt.initEvent("change", false, true);
 
             $.each(addressMapping, function (key, val) {
+                if (val.length === 0) {
+                    // no matching element in magento address form, eg street line 3 or more - ignoring value
+                    continue;
```

Comment:
I think this needs to `return true` according to the jQuery docs? jQuery each is different from JS iterations.